### ISSUE TITLE
Quick Format: Right-arrow from focused bubble selects accept button

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#4028]
1. Enable VoiceOver
2. Add an item to the citation
3. Press left-arrow to focus the item
4. Press right-arrow, with the intention of typing to search for another item

Normally this focuses the blank space after the bubble, but with VoiceOver enabled, the accept button gets focused and announced. You can technically still type to search, but it's not clear.

Not sure if there's a way to fix this.

**Comment by abaevbog** [comment]
Yeah, voiceover with its default setting has Quick Nav enabled. The [docs](https://support.apple.com/guide/voiceover/with-quick-nav-vo27943/10/mac/14.0) indicate that in this case, the arrow keypresses are treated as VO+arrow shortcuts which essentially takes over the regular arrow keypress events. This bypasses whatever listeners we have and moves focus mainly based on the DOM, so in this scenario our `keypress` handling doesn't even fire. This can be observed not only in the QF but pretty much everywhere else in the main app: for example, voiceover will move focus onto the "New Note" button in the toolbar on `arrowDown` on the magnifying glass of quicksearch, which is not how our focus navigation is setup.


To avoid this behavior, one can:
- do CMD-arrowUp/Down/Left/Right. This allows our listeners to handle focus for that keypress (this seems to be common knowledge though I can't find where it says that in the docs)
- simply turn off Quick Nav (VoiceOver Utility > Commanders > Quick Nav >  uncheck Enable Quick Nav). If Quick Nav is unchecked, this does not happen for me

I think it might be worth to add this somewhere into the docs for reference...


For this specific case, we could prevent voiceover from focusing the button by marking it as `aria-disabled` or setting `role="none"` (since the button doesn't need to be focusable at all). But even then, left/right arrows will jump focus between the bubbles without allowing our listeners to create and focus the inputs in between, so I think one would have to avoid the Quick Nav focus handling in this way or another to properly use the QF. If there is a lot of confusion about it, I suppose we could consider adding a note about Quick Nav into the `aria-description` strings.
